target/snitch_cluster/sw/snax/streamer-gemm-conv-simd/src/snax-streamer-gemm-conv-simd-lib.c

```diff
@@ -154,6 +154,10 @@ void set_gemmx_streamer_csr(
     csrw_ss(T_STRIDE_READER_WRITER_1_0, D32tlstride0);
     csrw_ss(T_STRIDE_READER_WRITER_1_1, D32tlstride1);
     csrw_ss(T_STRIDE_READER_WRITER_1_2, D32tlstride2);
+
+    // set the transpose
+    csrw_ss(TRANSPOSE_CSR_READER_0, transpose_A == 0 ? 1 : 0);
+    csrw_ss(TRANSPOSE_CSR_READER_0, transpose_B == 0 ? 1 : 0);
 }
```

Comment:
This just sets the same csr twice, instead of `TRANSPOSE_CSR_READER_1`, how is it possible this is not detected in the tests?

Reply:
Modified! The transpose is enabled by default and the test needs it enabled so this typo is not detected.